src/SFA.DAS.AANHub.Data/Repositories/MembersReadRepository.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Microsoft.EntityFrameworkCore;
+using SFA.DAS.AANHub.Domain.Entities;
+using SFA.DAS.AANHub.Domain.Enums;
+using SFA.DAS.AANHub.Domain.Interfaces.Repositories;
+using System.Diagnostics.CodeAnalysis;
+
+namespace SFA.DAS.AANHub.Data.Repositories
+{
+    [ExcludeFromCodeCoverage]
+    internal class MembersReadRepository : IMembersReadRepository
+    {
+        private readonly AanDataContext _aanDataContext;
+
+        public MembersReadRepository(AanDataContext aanDataContext) => _aanDataContext = aanDataContext;
+
+        public async Task<Member?> GetMember(Guid? id) => await _aanDataContext
+                .Members
+                .AsNoTracking().Where(m => m.Id == id && m.Status == MembershipStatuses.Live.ToString())
```

Comment:
I may not have been part of some conversations, but would we not want to get the member regardless of the status?

Reply:
Since this is the member who is initiating the action, I think it is right to check if the member is active.

Reply:
Is this not also to be used as a general lookup? Potentially a default bool? Something like
`public async Task<Member?> GetMember(Guid? id, bool includeAllMembers = false)`

Reply:
Potentially, but we don't have that requirement yet. When we have that use case, we will end up doing something you have suggested, but I don't want to assume that now.

Reply:
We could remove the check from repository and add it to the validator